datasets/za/mm_executive_mayors/crawler.py

```diff
@@ -0,0 +1,105 @@
+# from followthemoney.helpers import post_summary
+from urllib.parse import unquote
+import re
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+ROLES_OF_INTEREST = [
+    "Mayor/Executive Mayor",
+    "Municipal Manager",
+]
```

Comment:
Let's include CFO and make it a set for more efficient lookups

```suggestion
ROLES_OF_INTEREST = {
    "Mayor/Executive Mayor",
    "Municipal Manager",
}
```